Users/Glob_Patterns.rst

```diff
@@ -192,13 +182,48 @@ brackets. Parentheses that have no match are ignored as well as
 
 ::
 
-    >>> fnmatch("abc", "a**c")
-    True
     >>> fnmatch("abbc", "a**c")
     True
     >>> fnmatch("a/c", "a**c")
     True
-    >>> fnmatch("a?c", "a**c")
-    True
-    >>> fnmatch("ac", "a**c")
-    True
+
+Using Glob Patterns
+-------------------
+
+Suppose you want `SpaceConsistencyBear` to perform an analysis on a file
+``first.c``. You should use the command:
```

Comment:
suppose You want `SpaceConsistencyBear` to perform an analysis on a file
+``first.c``. you should use the command:

Like this ? I didn't get the small and capital letters' combination.